Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkDemos/SR-ResearchDemos/StereoPicture/EyeLink_StereoPicture.m

```diff
@@ -0,0 +1,408 @@
+                imgData = imread(imgNameRight); % Read image from file
+            end
+            imgTexture(itScr+1) = Screen('MakeTexture',window, imgData); % Convert image file to texture
+            Screen('DrawTexture', window, imgTexture(itScr+1)); % Prepare image texture on backbuffer
+            Screen('TextSize', window, 30); % Specify text size
+            Screen('DrawText', window, 'Press space bar to end trial', 5, height-35, 0); % Prepare text on backbuffer            
+        end
+        [~, RtStart] = Screen('Flip', window); % Present stimulus
+        % Write message to EDF file to mark the start time of stimulus presentation.
+        Eyelink('Message', 'STIM_ONSET');       
+        % Write !V IMGLOAD message to EDF file: provides instructions for DataViewer so it will show trial stimulus as backdrop
+        % See DataViewer manual section: Protocol for EyeLink Data to Viewer Integration > Image Commands
+        Eyelink('Message', '!V IMGLOAD CENTER %s %d %d', imgNameLeft, width/2, height/2);  
+        % Write !V IAREA message to EDF file: creates interest area around image in DataViewer
+        % See DataViewer manual section: Protocol for EyeLink Data to Viewer Integration > Interest Area Commands
+        Eyelink('Message', '!V IAREA RECTANGLE %d %d %d %d %d %s', 1, round(width/2-imgInfo.Width/2), round(height/2-imgInfo.Height/2), round(width/2+imgInfo.Width/2), round(height/2+imgInfo.Height/2),'IMAGE_IA');
+        
+        % STEP 5.4: WAIT FOR KEYPRESS; SHOW BLANK SCREEN; STOP RECORDING
+        
+        while 1 % loop until error or space bar press
+            % Check that eye tracker is  still recording. Otherwise close and transfer copy of EDF file to Display PC
+            err = Eyelink('CheckRecording');
+            if(err ~= 0)
+                fprintf('EyeLink Recording stopped!\n');
+                % Transfer a copy of the EDF file to Display PC
+                Eyelink('SetOfflineMode'); % Put tracker in idle/offline mode
+                Eyelink('CloseFile'); % Close EDF file on Host PC
+                Eyelink('Command', 'clear_screen 0'); % Clear trial image on Host PC at the end of the experiment
+                WaitSecs(0.1); % Allow some time for screen drawing
+                % Transfer a copy of the EDF file to Display PC
+                transferFile; % See transferFile function below
+                cleanup; % Abort experiment (see cleanup function below)
+                return
+            end
+            % End trial if spacebar is pressed
+            [~, RtEnd, keyCode] = KbCheck;
+            if keyCode(spaceBar)
+                % Write message to EDF file to mark the spacebar press time
+                Eyelink('Message', 'KEY_PRESSED');
+                reactionTime = round((RtEnd-RtStart)*1000); % Calculate RT from stimulus onset
+                break;
+            end
+        end % End of while loop
+        
+        % Draw blank screen at end of trial

```

Comment:
This drawing is not strictly needed. A simple ``Screen('Flip', window);`` as in line 314 will do.
Reason: As the previous flip in your stereo image presentation code did not specify a 'dontclear' parameter, and thereby used the default of 0 == Do clear after Flip, the framebuffer was already automatically cleared to background color. So a simple Flip will now display that blank background colored image.